challenges/objects-arrays.js

```diff
@@ -49,21 +74,28 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+let universities = graduates.map(graduate => {
+  return graduate.university;
+});
+universities = universities.sort();
```

Comment:
You could have just called universities.sort() without reassigning universities to itself.